File: hw/Pérez/U2/HW18ZooJavaScript/model/Food.js
export class Food {
    constructor(id, description) {
        this.id = id;
        this.description = description;
    }

    getId() {
        return this.id;
    }

    setId(id) {
        this.id = id;
    }

    getDescription() {
        return this.description;
    }

    setDescription(description) {
        this.description = description;
    }

    toString() {
        return `Food{id=${this.id}, description=${this.description}}`;
    }
}

File: ws/ayuquina/u2/ws21zooWithInheritence/Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Food.java
package ec.edu.espe.zoo.model;

/**
 *
 * @author Danny Ayuquina, LogicLegion, DCCO-ESPE
 */
public class Food {
    private int id;
    private String description;

    public Food(int id, String description) {
        this.id = id;
        this.description = description;
    }

    @Override
    public String toString() {
        return "Food{" + "id=" + id + ", description=" + description + '}';
    }

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }
    
}
